src/query/storages/system/src/tables_table.rs

```diff
@@ -409,7 +446,7 @@ where TablesTable<T, U>: HistoryAware
                             catalogs.push(ctl_name.as_str());
                             databases.push(db_name.to_owned());
                             database_tables.push(table);
-                            if ownership.is_empty() {
+                            if ownership.is_empty() || !get_ownership {
```

Comment:
`!get_ownership` can remove, if `get_ownership` is false, `ownership` is always empty.

Reply:
Done